mojaloop-simulator/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "1.0"
+appVersion: "9.4.10"
 description: Helm Chart for the Mojaloop (SDK based) Simulator
 name: mojaloop-simulator
-version: 2.0.0
+version: 3.0.0
```

Comment:
I have seen different approaches across different teams... So whats the difference between appVersion and version?

Reply:
My understanding was that `appVersion` was the version of the enclosed application, i.e. the simulator version, and that `version` was the version of the chart. Although, it's tricky in this case because the sim and the SDK are probably versioned differently, so (if I'm correct about the usage of `appVersion`) `appVersion` should really be composite. So perhaps I shouldn't have changed it.

I did update `version` because this is a breaking change to the chart.

Reply:
I reverted the change to `appVersion`.